cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/BaseCommandTest.java

```diff
@@ -61,6 +68,16 @@ public void setup() throws IOException {
         this.printStream = new PrintStream(this.console);
     }
 
+    @BeforeSuite
+    public void copyBalTools() throws IOException {
+        // copy the bal-tools.toml to <user-home>/.ballerina/.config/
+        Path balToolsTomlSrcPath = Paths.get("src/test/resources/test-resources/" +
```

Comment:
@azinneera Shall we use path separator instead of `/`?

Reply:
The `Path.get` api takes care of the OS when constructing the path.